Makefile

```diff
@@ -22,7 +22,7 @@ stop: ## Stop the project
 	$(DOCKER_COMPOSE) down
 
 lint: ## Linter
-	$(DOCKER_COMPOSE) exec -T server prospector
+	$(DOCKER_COMPOSE) exec -T server python3 -m prospector --profile /code/.prospector.yaml
```

Comment:
@BilelJegham does it break the tool chain?

Reply:
Tu te souviens le problème me disant que docker. Je prenais pas en compte la config propector. Voici le correctif 🙂